bootstraptest/test_ractor.rb

```diff
@@ -1987,3 +1987,127 @@ def require feature
   GC.start
   :ok.itself
 }
+
+# moved objects being corrupted if embeded (String)
+assert_equal 'ok', %q{
+  ractor = Ractor.new { Ractor.receive }
+  obj = "foobarbazfoobarbazfoobarbazfoobarbaz"
+  ractor.send(obj.dup, move: true)
+  rountriped_obj = ractor.take
```

Comment:
I think this is a typo? It should be `roundtripped_obj`. All of the tests have this typo.

```suggestion
  roundtripped_obj = ractor.take
```